src/Admin/Vue.php

```diff
@@ -27,6 +27,9 @@ public function init() {
 			$logged = $this->payplug_section_logged();
 			$payplug_wooc_settings = get_option( 'woocommerce_payplug_settings', [] );
 			unset($payplug_wooc_settings["payplug_live_key"]);
+			unset($payplug_wooc_settings["payplug_test_key"]);
+			unset($payplug_wooc_settings["payplug_password"]);
+			unset($payplug_wooc_settings["payplug_merchant_id"]);
```

Comment:
we need the merchant id for the one click and save card

Reply:
This will prevent sending the mercahnt _id to vue, and it is not used there.